bob-common/src/mapper.rs

```diff
@@ -179,23 +179,27 @@ impl Virtual {
             .collect()
     }
 
-    pub fn get_operation(&self, key: BobKey) -> (VDiskId, Option<DiskPath>) {
+    pub fn get_operation(&self, key: BobKey) -> (VDiskId, Option<Vec<DiskPath>>) {
         let virt_disk = self.get_vdisk_for_key(key).expect("vdisk not found");
-        let disk = virt_disk.replicas().iter().find_map(|disk| {
+        let disks: Vec<DiskPath> = virt_disk.replicas().iter().filter_map(|disk| {
```

Comment:
Most of the time we will have 0 or 1 paths in the result. That's why I suggest to use regular for-loop. Initially, result should be `None` and on the first match it should be converted into `Some(Vec::with_capacity(1))`